service/hook/gogs/gogs.go

```diff
@@ -61,19 +69,11 @@ func detectContentTypeAndEventID(header http.Header) (string, string, error) {
 	return contentType, eventID, nil
 }
 
-func transformCodePushEvent(codePushEvent CodePushEventModel) hookCommon.TransformResultModel {
-	if !strings.HasPrefix(codePushEvent.Ref, "refs/heads/") {
```

Comment:
This could be kept, related to https://github.com/bitrise-io/bitrise-webhooks/pull/43/files#r101161888